opt/definitions.rs

```diff
@@ -139,6 +139,10 @@ pub struct Dominators {
     /// The maximum number of rows, regardless of depth in the tree, to display.
     #[structopt(short = "r")]
     max_rows: Option<u32>,
+
+    /// The name of the function whose dominator subtree should be printed.
+    #[structopt(long = "function", default_value = "")]
+    func_name: String,
```

Comment:
Can this be an optional positional argument, ie `twiggy dominators input.wasm my_function`? Unsure if that will be easy to do in structopt or not, but if we can do it without too much hassle, then I think that would be a little nicer ergonomics for users.

I'd also name this "subtree" rather than "func_name".